### DISCUSSION TITLE
More dynamic logging console

### DISCUSSION THREAD

**Discussion by curruwilla** [laravel/framework#43500] [Ideas]
Hello

Is it possible to include this package to extend the commands to display information in the console?
This would make it much easier to manipulate and organize what is displayed.

https://github.com/curruwilla/console-pretty-log

I use the console a lot and ended up creating it because it was a necessity to better visualize the information. And the package was so dynamic in the customization that it will work for anyone :)

**Comment by henzeb** [comment]
They have just given the console command output an update. they use a package developed by @nunomaduro. https://github.com/nunomaduro/termwind

**Reply by curruwilla** [reply]
Wow, I didn't know, I'll test it
thanks
